.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        include:
+          - cuda: 10.2.89
+            gcc: 8.5.0
+            nvcc: 10.2
+            python: 3.8
+            pytorch: 1.7.1
+          - cuda: 11.2.2
+            gcc: 10.3.0
+            nvcc: 11.2
+            python: 3.9
+            pytorch: 1.8.0
```

Comment:
Should this be 1.9?  That's the most recent release.

Reply:
TorchANI has a problem with PyTorch 1.9.